src/source/Sdp/Deserialize.c

```diff
@@ -1,37 +1,48 @@
-#define LOG_CLASS "SDP"
+#define LOG_CLASS "SDPDeserialize"
 #include "../Include_i.h"
+#include "kvssdp/sdp_deserializer.h"
 
-STATUS parseMediaName(PSessionDescription pSessionDescription, PCHAR pch, UINT32 lineLen)
+STATUS parseMediaName(PSessionDescription pSessionDescription, PCHAR mediaValue, SIZE_T mediaValueLength)
 {
     ENTERS();
     STATUS retStatus = STATUS_SUCCESS;
+    SIZE_T copyLength;
```

Comment:
copyLength - Can we make the variable name more meaningful, like mediaLength or something similar.

Reply:
Updated